libs/czmq/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+## Copyright (C) 2019 CZ.NIC z.s.p.o. (http://www.nic.cz/)
+#
+## This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+# #
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=czmq
+PKG_VERSION:=4.1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/zeromq/czmq/tar.gz/v$(PKG_VERSION)?
```

Comment:
This project have release tarballs, please use those instead.

Reply:
Changed to release tarball